compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/ProjectDirs.java

```diff
@@ -83,4 +83,50 @@ public static boolean containsSourceFiles(Path pkgPath) {
         }
         return sourceFiles.size() > 0;
     }
+
+    /**
+     * Checks if the source is from tests.
+     * @param sourcePath source path
+     * @param sourceRoot source root path
+     * @param pkg package name
+     * @return true if its a test source, else false
+     */
+    public static boolean isTestSource(Path sourcePath, Path sourceRoot, String pkg) {
+        // Check if it the package name is "." i.e. if its a single ballerina source file, if so it should be added to
+        // the bLangPackage
+        if (pkg.equals(Names.DOT.value)) {
```

Comment:
As a best practice, have the static value as the first expr.

Reply:
Changed the static value to be the first expression when checking for the equality with 4e9c501